src/utils/jQueryMock.js

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
remove comment? honestly i dont know whats the policy on copyright comments but we dont use them in other files